gradle/libs.versions.toml

```diff
@@ -59,12 +60,17 @@ picasso = "com.squareup.picasso:picasso:_"
 recyclerview = "androidx.recyclerview:recyclerview:_"
 retrofit = "com.squareup.retrofit2:retrofit:_"
 retrofitMoshi = "com.squareup.retrofit2:converter-moshi:_"
-retrofitRxJava = "com.squareup.retrofit2:adapter-rxjava2:_"
+retrofitRxJava2 = "com.squareup.retrofit2:adapter-rxjava2:_"
+retrofitRxJava3 = "com.squareup.retrofit2:adapter-rxjava3:_"
+coil = "io.coil-kt:coil:_"
```

Comment:
why add coil?

Reply:
this one I added as part of rxjava demo, which is still in progress